ext/reflection/php_reflection.c

```diff
@@ -627,6 +630,32 @@ static void _const_string(string *str, char *name, zval *value, char *indent)
 }
 /* }}} */
 
+/* {{{ _class_const_string */
+static void _class_const_string(string *str, char *name, zend_class_constant *c, char *indent)
+{
+	char *type = zend_zval_type_name(&c->value);
+	char *visibility = NULL;
+	zend_string *value_str = zval_get_string(&c->value);
```

Comment:
Would be nice to reuse the https://github.com/dstogov/php-src/blob/bug-69980-class-constants/ext/reflection/php_reflection.c#L729 code instead. String cast usually has bad representation.